src/gdcdictionary/schemas/treatment.yaml

```diff
@@ -160,6 +160,39 @@ properties:
     type: number
     minimum: 0
 
+  margin_distance:
+    $ref:
+      - "_terms.yaml#/margin_distance/common"
+    type: number
+    minimum: 0
```

Comment:
should we add "minimum: 0" to the Jira task?

Reply:
added 👍